resources/user-agents/apps/openoffice.json

```diff
@@ -11,29 +11,19 @@
         "Comment": "OpenOffice",
         "Browser": "OpenOffice",
         "Browser_Type": "Application",
-        "Browser_Maker": "The Apache Software Foundation",
-        "Frames": "true",
-        "IFrames": "true",
-        "Tables": "true",
-        "Cookies": "true",
-        "JavaScript": "true",
-        "CssVersion": "2"
+        "Browser_Maker": "The Apache Software Foundation"
       },
       "children": [
         {
           "match": "OpenOffice/4.0*",
           "properties": {
-            "Version": "4.0",
-            "MajorVer": "4",
-            "MinorVer": "0"
+            "Version": "4.0"
```

Comment:
The definition of the version could be factorized ?  
Could the versions list be handled at the level of the children instead of the main ?